vsg/rules/process/rule_036.py

```diff
@@ -0,0 +1,16 @@
+from vsg.rules import prefix_rule
+from vsg import utils
+
+
+class rule_036(prefix_rule):
+    '''
+    Process rule 036 checks for prefixes in process labels.
+    '''
+
+    def __init__(self):
+        prefix_rule.__init__(self, 'process', '036', 'isProcessLabel')
+        self.prefixes = ['proc_']
+        self.solution = 'Process'
```

Comment:
So I noticed the `_get_solution` method in the `prefix_rule` class shows the following:

```python
    def _get_solution(self, iLineNumber):
        return self.solution + ' identifiers should start with one of ' + str(self.prefixes) + ' prefixes.'
```
With `self.solution` equal to `Process`, the solution printed out would be:
```mono
Process identifiers should start with one of proc_ prefixes
```
Which seems misleading.
If you could make the following updates:
  1)  Change `self.solution` to `Process label`
  2)  Remove `identifiers` from the above code in `prefix_rule.py`
  3)  Find the other rules using `prefix_rule` and add `identifiers` to their `self.solution` attribute

I think that should clarify what the user should fix.

Reply:
I adapted each solution of a prefix rule to the corresponding docstring. The solutions should be more accurate now.